src/pages/deprecated.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Deprecated
+decscription: Deprecation info
+---
+
+# This CosmWasm documentation is deprecated
+
+This CosmWasm documentation was not updated for a long time, and it got deprecated. Please refer to other sources:
+
+* [CosmWasm Book](https://book.cosmwasm.com/)
+* [Sylvia Book](https://cosmwasm.github.io/sylvia-book/index.html)
+* [CosmWasm Academy](https://academy.cosmwasm.com/)
+* [awesome-cosmwasm](https://github.com/CosmWasm/awesome-cosmwasm)
+* [cosmwasm-std documentation](https://docs.rs/cosmwasm-std/)
```

Comment:
I created a tag for the last version before this PR. Could you add a link to the source code to the note?
https://github.com/InterWasm/docs/tree/archive

Reply:
Yup, I will add.